include/GafferSceneUI/CropWindowTool.h

```diff
@@ -80,8 +81,8 @@ class GAFFERSCENEUI_API CropWindowTool : public GafferUI::Tool
 		void preRender();
 
 		void findCropWindowPlug();
-		bool findCropWindowPlug( GafferScene::ScenePlug *scene, bool enabledOnly  );
-		bool findCropWindowPlugFromNode( GafferScene::ScenePlug *scene, bool enabledOnly  );
+		bool findCropWindowPlug( const GafferScene::SceneAlgo::History *history, bool enabledOnly  );
+		bool findCropWindowPlugFromNode( const GafferScene::SceneAlgo::History *history, bool enabledOnly  );
```

Comment:
Why change the signature here? This function doesn't need access to the history does it? We shouldn't give it more privilege that it needs.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3207/commits/c4cde15814d517eba484859df2b22169279dedd4